config/a8c-for-agencies-stage.json

```diff
@@ -39,6 +39,7 @@
 		"a4a-pressable-referrals": true,
 		"a4a-updated-add-new-site": true,
 		"a4a-signup-v2": false,
+		"a4a-signup-v2-via-email": true,
```

Comment:
To make testing easier, we only need to worry about one feature flag, which we set this one to true by default. This feature flag also relies on `a4a-signup-v2`, so there is no need to worry about this staging interruption.